src/proto/parameters.proto

```diff
@@ -435,3 +436,24 @@ message PossessionTrackerConfig
         (bounds).max_double_value = 10.0
     ];
 }
+
+message CreaseDefenderConfig
+{
+    /*
+    Max ratio between distances (crease and ball) / (crease and nearest enemy) for
+    crease to chase ball. Scale from (0, 1)
+    ┌──────────────────────────────────────────┐
+    | Crease | <───────────────────────> Enemy |
+    |        | <────> Ball                 |   |
+    |       ()         x                  ()   |
+    └──────────────────────────────────────────┘
+     */
+    required double max_get_ball_ratio_threshold = 1
+        [default = 0.3, (bounds).min_double_value = 0.0, (bounds).max_double_value = 1.0];
+    // Max distance that the crease will try and get possession of a ball
+    required double max_get_ball_radius_m = 2
+        [default = 1, (bounds).min_double_value = 0.0, (bounds).max_double_value = 10.0];
+    // Max speed of ball that crease will try and get possession
+    required double max_ball_speed_to_get_ms = 3
+        [default = 0.5, (bounds).min_double_value = 0.0, (bounds).max_double_value = 7.0];
```

Comment:
nit: for consistency, trailing `ms` is used for milliseconds. For meters per second we use trailing `m_per_s`